src/ziggurat/config.clj

```diff
@@ -232,18 +244,55 @@
   {:enabled true
    :ssl-keystore-location <>
    :ssl-keystore-password <>
+   :mechanism <>
+   :protocol <>
     {:jaas {:username <>
             :password <>
-            :mechanism <>}}}
+            :login-module <>}}}
   "
   (let [ssl-configs-enabled (:enabled ssl-config-map)
-        jaas-config         (get ssl-config-map :jaas)]
-    (if (true? ssl-configs-enabled)
+        jaas-config         (get ssl-config-map :jaas)
+        mechanism           (get ssl-config-map :mechanism)
+        protocol            (get ssl-config-map :protocol)
+        login-callback-handler (get ssl-config-map :login-callback-handler)]
```

Comment:
please add what this is doing in the comments above here for context.

Reply:
added above.